packages/dx-react-scheduler-material-ui/src/templates/date-navigator/calendar/table.jsx

```diff
@@ -13,28 +14,49 @@ const styles = {
   },
 };
 
-export const TableBase = ({
+const TableBase = ({
   rowComponent: Row,
   cellComponent: Cell,
+  headerRowComponent: HeaderRow,
+  headerCellComponent: HeaderCell,
   classes,
   className,
   cells,
+  headerCells,
+  onCellClick,
   ...restProps
 }) => (
   <TableMUI
     className={classNames(classes.table, className)}
     {...restProps}
   >
+    <TableHead>
+      <HeaderRow>
+        {headerCells.map((cell) => {
+          const value = moment(cell).format('ddd');
+          return (
+            <HeaderCell
+              key={`header_${value}`}
```

Comment:
Is it necessary? I think that `header_${value}` string concatenation is excess in this context

Reply:
I don't like an idea that templates calculates keys. It should be done on the core side if possible